Makefile.Common

```diff
@@ -4,7 +4,7 @@ SRC_ROOT := $(realpath $(dir $(lastword $(MAKEFILE_LIST))))
 # build tags required by any component should be defined as an independent variables and later added to GO_BUILD_TAGS below
 GO_BUILD_TAGS=""
 GOTEST_OPT?= -race -v -timeout 300s --tags=$(GO_BUILD_TAGS)
-GOTEST_INTEGRATION_OPT?= -race -timeout 120s
+GOTEST_INTEGRATION_OPT?= -race -timeout 900s
```

Comment:
Did you add a test that runs 15 minutes?

Reply:
Yes. This is to allow the Oracle DB docker image to be created, which takes about 10 minutes on a local machine. 

There is no published supported Oracle DB docker image, the supported method of creating docker images is to download a specific repository, and run a script. This is the `Setup Oracle DB Docker Image Dependency` step I added in the `build-and-test` workflow.

Reply:
What about this one https://container-registry.oracle.com/ords/f?p=113:4:114145384398230:::4:P4_REPOSITORY,AI_REPOSITORY,AI_REPOSITORY_NAME,P4_REPOSITORY_NAME,P4_EULA_ID,P4_BUSINESS_AREA_ID:8,8,Oracle%20Database%20Standard%20Edition%202,Oracle%20Database%20Standard%20Edition%202,1,0&cs=35MRG-gBMZGZd7cPIsIXl-GsL7HDV1AQqJDptQwn_G5F0noHF73yy9RusjZsCAmt1Zfmi3Cgq9sTMn5HXGtIuhw ?

Reply:
Otherwise please make sure that the image is cached on the CI. Additional 15 minutes to every CI run is a lot.

Reply:
The link you included worked, that's a much nice way to handle this. Thanks for finding that, not sure how I didn't find it in my search!

I've updated as requested, and reverted the timeout to the original 120s.